ngrinder-controller/src/main/java/org/ngrinder/user/service/UserService.java

```diff
@@ -108,13 +112,13 @@ public User getOne(String userId) {
 	 * @return user
 	 */
 	@Transactional
+	@Cacheable(value = CACHE_USER_WITH_FOLLOWERS, key = "#userId")
 	public User getOneWithFollowers(String userId) {
-		User one = userRepository.findOneByUserId(userId);
-		one.getFollowers().size();
-		return one;
+		User user = userRepository.findOneByUserId(userId);
+		Hibernate.initialize(user.getFollowers());
```

Comment:
what about doing this in ```getOne```

Reply:
```Hibernate.initialize(user.getFollowers());``` this code looks the same as findOne.

http://blog.naver.com/PostView.nhn?blogId=kbh3983&logNo=221170946678

maybe getOne could EntityNotFoundException occur?

Reply:
@junoyoon @songeunwoo 

Making `getOne()` as eager(followers) may affect other usage points, but it seems not problem.
it's better to have fewer changes than before.